docs/API_docs/UserInfoAPI.md

```diff
@@ -27,51 +27,81 @@ Possible return message:
 
 ```
 {
-    "_id": {
-        "$oid": "5e43132fe3fc7eea4a59b5a4"
-    },
-    "username": "Arianne",
+    "username": "Sebastien",
+    "bio": "Welcome to mypanda space!!",
+    "nb_pictures": 5,
     "pictures": [
         {
-            "_id": {
-                "$oid": "5e431352e3fc7eea4a59b5a5"
-            },
-            "date": {
-                "$date": 1581436162806
-            },
-            "owner": "Arianne",
-            "user": "Arianne",
-            "link": "picture 1",
-            "message": "Awesome",
+            "owner": "Sebastien",
+            "user": "Sebastien",
```

Comment:
We should only keep one since we no longer need repost

Reply:
Yep...

Reply:
done